### DISCUSSION TITLE
Android Keyboard issues

### DISCUSSION THREAD

**Discussion by 2Senn** [infinitered/ignite#2482] [Q&A]
Hey everyone, so if I have a form inside the screen component, the keyboardavoidingview does not really work. The keyboard just goes above the input field and you need to manually scroll down. This is an android specific issue.

Would be great if someone can help!

Thanks

**Comment by Abdulmateenchitrali** [comment]
Set android:windowSoftInputMode="adjustPan" in your manifest file, and it will work as you expect.

E.g.

```
<application
  android:name=".MainApplication"
  android:allowBackup="true"
  ...
  <activity
    android:name=".MainActivity"
    android:label="@string/app_name"
    ...
    android:windowSoftInputMode="adjustPan">
    ...
  </activity>
  ...
</application>
```
